virtool/subtractions/db.py

```diff
@@ -182,3 +182,21 @@ def lookup_nested_subtractions(
             }
         },
     ]
+
+
+async def get_subtraction_names(subtraction_ids: List, db) -> List[dict]:
```

Comment:
1. Let's upgrade the signature of this function.
     ```suggestion
     async def get_subtraction_names(mongo: "Mongo", subtraction_ids: List[str]) -> List[str]:
     ```

2. Refactor this function to be more efficient. It likely possible to use `Mongo.find` to massively improve perf. If not, `asyncio.gather()` is the less desirable option.
3. The docstring is not accurate.
4. The return type can be more specific.